### ISSUE TITLE
Nextcloud LXC Alpine Linux Imagick needed

### ISSUE THREAD

**Issue by eakteam** [tteck/Proxmox#2628]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

The default installation works fine but if we go to theme options of NextCloud and trying to change some things, and rebooting the server, NextCloud cannot be accessed anymore. I have searched a lot about it and seems to be a issue with php-imagick extension that doesn't exist....

### Which Linux distribution are you employing?

Default Settings

**Comment by tteck** [comment]
Have you attempted adding the `php-imagick` extension, if available, to see if that resolves your problem?

**Comment by eakteam** [comment]
> Have you attempted adding the `php-imagick` extension, if available, to see if that resolves your problem?

No, because I don't know exactly how to do that on Alpine and currently can't go throught researching... But tried to reinstall it and see if it happens exactly by that. 

At some point I made some modifications which don't remember all of them and stucked at login page where cannot login anymore. No logs, no error nothing. Just looping at the login page.

Now I have reinstalled and for each change will restart and see if I can reproduce it again...

Anyway, imagick is needed as NextCloud claims if we have to do with some theming options or other related stuff....
